nevergrad/optimization/oneshotcalais.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional, Union
+import numpy as np
+from scipy import stats
+from ..common.typetools import ArrayLike
+from ..instrumentation import Instrumentation
+from . import sequences
+from . import base
+from .oneshot import *
+
+MetaCalais = SamplingSearch(
```

Comment:
at some point not too far aways, these should be mostly hidden from main users, it makes the list of algorithms very "noisy", why do we need so many? If it's for Xp we could create them on demand